src/api/firebase.js

```diff
@@ -144,13 +144,15 @@ export async function createList(userId, userEmail, listName) {
 export async function shareList(listPath, currentUserId, recipientEmail) {
 	// Check if current user is owner.
 	if (!listPath.includes(currentUserId)) {
+		throw new Error('The list is not yours to share.');
 		return;
 	}
 	// Get the document for the recipient user.
 	const usersCollectionRef = collection(db, 'users');
 	const recipientDoc = await getDoc(doc(usersCollectionRef, recipientEmail));
 	// If the recipient user doesn't exist, we can't share the list.
 	if (!recipientDoc.exists()) {
+		throw new Error('User does not yet exist.');
```

Comment:
praise/suggestion: I like this approach for passing error/alert messages to the UI! We could think about making that consistent across functions if possible later on (e.g., the createList function Andrea and I worked on)